admin_manual/groupware/calendar.rst

```diff
@@ -76,6 +76,19 @@ If you need to allow this, change the following parameter to::
 
  php occ config:app:set dav webcalAllowLocalAccess --value yes
 
+Trash bin
+---------
+
+Nextcloud supports a calendar, events and tasks trash bin.
```

Comment:
@ChristophWurst when will we change the trash bin name to Deleted Items? If at all?

Reply:
Good comment. That would be align it to the Files app, there it is called "Deleted files".

Reply:
We might do that but right now the UI says *trash bin*. So the documentation should be consistent. If we rename then docs should only change for 23+ or whatever versions will be affected

Reply:
Oh, my thought was to change this in UI as well. It is a relative new feature and therefore it is easy to change I think.
Jm2c.